packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -464,6 +464,7 @@ export function toDocumentSymbol(info: model.DocumentSymbol): theia.DocumentSymb
         toRange(info.selectionRange),
     );
     if (info.children) {
+        // tslint:disable-next-line:no-any
         result.children = info.children.map(toDocumentSymbol) as any;
```

Comment:
This one is a bit suspicious.  It would be nice to have a comment to say why it's needed.

Reply:
sure, I'll ping the developer.
@azatsarynnyy why was `any` used in this case?

Reply:
@vince-fugnitto I guess the entire function can be removed since it's not used anymore. If you'd like I submit a PR for that just let me know? Or you can do it within that PR

Reply:
> @vince-fugnitto I guess the entire function can be removed since it's not used anymore. If you'd like I submit a PR for that just let me know? Or you can do it within that PR

sure I'll take care of it and add you as a reviewer